{
    "type": "bundle",
    "id": "bundle--2a63de83-2571-4de8-a9ea-03de8ff1bc08",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--a92afd39-7b76-4729-8e60-c901c4389148",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-05-22T00:29:02.106683Z",
            "modified": "2024-05-22T00:29:02.106683Z",
            "name": "CVE-2021-47291",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\nipv6: fix another slab-out-of-bounds in fib6_nh_flush_exceptions\n\nWhile running the self-tests on a KASAN enabled kernel, I observed a\nslab-out-of-bounds splat very similar to the one reported in\ncommit 821bbf79fe46 (\"ipv6: Fix KASAN: slab-out-of-bounds Read in\n fib6_nh_flush_exceptions\").\n\nWe additionally need to take care of fib6_metrics initialization\nfailure when the caller provides an nh.\n\nThe fix is similar, explicitly free the route instead of calling\nfib6_info_release on a half-initialized object.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2021-47291"
                }
            ]
        }
    ]
}